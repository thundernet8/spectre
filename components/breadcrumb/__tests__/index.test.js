import React from "react";
// import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import BreadCrumb from "..";

const BreadCrumbItem = BreadCrumb.Item;

describe("BreadCrumb", () => {
    it("should render BreadCrumb", () => {
        // TODO:
    });

    it("renders correctly", () => {
        const tree = renderer
            .create(
                <BreadCrumb>
                    <BreadCrumbItem>Home</BreadCrumbItem>
                    <BreadCrumbItem>Posts</BreadCrumbItem>
                    <BreadCrumbItem>One post name</BreadCrumbItem>
                </BreadCrumb>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
