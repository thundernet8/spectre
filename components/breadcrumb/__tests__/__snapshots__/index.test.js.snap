// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadCrumb renders correctly 1`] = `
<div
  className="spt-breadcrumb"
  style={undefined}
>
  <span>
    <span
      className="spt-breadcrumb-link"
    >
      Home
    </span>
    <span
      className="spt-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <span
      className="spt-breadcrumb-link"
    >
      Posts
    </span>
    <span
      className="spt-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <span
      className="spt-breadcrumb-link"
    >
      One post name
    </span>
    <span
      className="spt-breadcrumb-separator"
    >
      /
    </span>
  </span>
</div>
`;
